Makefile

```diff
@@ -96,7 +97,7 @@ build/%.owl: build/intermediate/properties.owl templates/%.tsv build/intermediat
 	template \
 	--template $(word 3,$^) \
 	--merge-before \
-	annotate --ontology-iri "https://purl.ihccglobal.org/$(notdir $@)" \
+	annotate --ontology-iri "https://purl.ihccglobal.org/$(notdir $@)" --version-iri "https://purl.ihccglobal.org/releases/$(TODAY)/$(notdir $@)" \
```

Comment:
I would like to follow the OBO convention, which would make it:

```
--version-iri "https://purl.ihccglobal.org/$(notdir $(basename $@))/$(TODAY)/$(notdir $@)"
```

Right?

Reply:
I thought of that, but won't this make it exceedingly hard to create a redirect? The suggestion I made had in mind a single redirect like this:

```
https://purl.ihccglobal.org/releases/ ---> https://github.com/IHCC-cohorts/data-harmonization/releases/download/v
```

If you would add the `$(notdir $(basename $@))` in the middle, you will have to create redirect rule for every single dictionary you create? Or is this something you will solve with some regex nginx hacking?

Reply:
The redirect rules won't be a problem.

Reply:
On second thought, please do

```
--version-iri "https://purl.ihccglobal.org/$(notdir $(basename $@))/releases/$(TODAY)/$(notdir $@)"
```

Reply:
Done! :)